chia/full_node/block_store.py

```diff
@@ -264,10 +263,10 @@ async def get_block_info(self, header_hash: bytes32) -> Optional[GeneratorBlockI
                     b.foliage.prev_block_hash, b.transactions_generator, b.transactions_generator_ref_list
                 )
 
-    async def get_generator(self, header_hash: bytes32) -> Optional[SerializedProgram]:
+    async def get_generator(self, header_hash: bytes32) -> Optional[bytes]:
         cached = self.block_cache.get(header_hash)
         if cached is not None:
-            return cached.transactions_generator
+            return None if cached.transactions_generator is None else bytes(cached.transactions_generator)
```

Comment:
This, and similar other instances below, is essentially what we do in `maybe_serialize` so perhaps we could move and reuse that helper more broadly?

Reply:
I only introduced that function in a test, I'm hesitant to depend on test code in production. And I'm also hesitant to turn this into a function here